### ISSUE TITLE
Accessibility issues on the REPL window: Toolbar

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2229]
![image](https://cloud.githubusercontent.com/assets/5177491/23572016/37c20cfe-0021-11e7-9c06-3fff8388032e.png)

**Comment by eclectir** [comment]
Error details-
DuplicateAttributes - Parent UIA_NameProperty - The element and its parent element must not have duplicated name property. Else the user will have difficulty to properly understand the control context.
https://msdn.microsoft.com/en-us/library/windows/desktop/ee684017(v=vs.85).aspx

MissingProperty - UIA_NameProperty - Element has no accessible name. If a user reaches an element with no name, then often they will have no way of knowing what the element relates to.
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationelement.automationelementinformation.name(v=vs.110).aspx
